examples/pennylane_run_advanced_usage.py

```diff
@@ -82,7 +82,7 @@ def circuit1(param):
 #
 # PennyLane uses the pattern that *all positional arguments to quantum functions are available
 # to be differentiated*, while *keyword arguments are never differentiated*. Thus, when using the
-# gradient-descent-based :ref:`optimizers <optimization_methods>` included in PennyLane, all
+# gradient-descent-based optimizers included in PennyLane, all
```

Comment:
needs to be changed in qml repo

Reply:
I actually propose to "deactivate" all links in the tuts for now. There are only of the order 5 or so, and they are not crucial. We can then do all the restructure, and activate them again.